File: development/NET/samples/server/ServerDLL/VB/OpcProject/MyAeAddressSpaceElement.vb
Imports System
Imports Softing.OPCToolbox.Server

Namespace DllSample
	Public Class MyAeAddressSpaceElement
		Inherits AeAddressSpaceElement
#Region "Constructors"
		'-------------------

		Public Sub New(ByVal aName As String, ByVal anUserData As System.UInt32, ByVal anObjectHandle As UInt32, ByVal aParentHandle As UInt32)
			MyBase.New(aName, anUserData, anObjectHandle, aParentHandle)
		End Sub

		Public Sub New()
		End Sub

		'--
#End Region
	End Class
End Namespace

File: development/NET/samples/server/ServerDLL/CS/OpcProject/MyAeAddressSpaceElement.cs
using Softing.OPCToolbox.Server;

namespace DllSample
{
	public class MyAeAddressSpaceElement : AeAddressSpaceElement
	{
		#region Constructors
		//-------------------

		public MyAeAddressSpaceElement(
			string aName,
			uint anUserData,
			uint anObjectHandle,
			uint aParentHandle)
			:
			base(
			aName,
			anUserData,
			anObjectHandle,
			aParentHandle)
		{
		}   //	end constructor

		public MyAeAddressSpaceElement()
		{
		}   //	end constructor

		//--
		#endregion
	}   //	end class MyAeAddressSpaceElemen
}
